src/pages/Home/Reviews/index.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { useMediaQuery } from "react-responsive";
+import Carousel from "../../../components/Carousel";
+import CarouselItem from "../../../components/Carousel/CarouselItem";
+import Title from "../../../components/Title";
+import { CarouselSlider_mockData } from "../../../assets/data/carouselSlider_mockData";
+import { app_mockData } from "../../../assets/data/app_mockData";
+
+const Reviews = () => {
+  const isTabletOrMobile = useMediaQuery({ query: "(max-width: 1200px)" });
+
+  const handlerScrollUp = () => {
```

Comment:
As fare as i know this can be done using css and html anchors, this solution is fine but you have to move it to utils